.gitignore

```diff
@@ -6,14 +6,10 @@
 /build
 /dist
 /MANIFEST
-/setuptools-3.1-py2.7.egg
-/setuptools-3.1.zip
 /stone.egg-info
 .arcconfig
 .DS_Store
 .idea
-parser.out
```

Comment:
`ply`'s a weird library all around. For example, you specify grammar rules in method docstrings. It's simply the libraries behavior to dump intermediate files into the cwd.